src/dtflw/lazy_notebook.py

```diff
@@ -82,14 +85,25 @@ def input(self, name: str, file_path: str = None, source_table: str = None) -> L
                 self.rel_path
             )
 
-        elif not self.ctx.storage.is_abs_path(input_file_path):
+        elif not self.ctx.storage.is_abs_path(input_file_path) and not input_file_path=="":
```

Comment:
Please add spaces: and not input_file_path == ""

Reply:
How about `and input_file_path != ""` ?

Reply:
@User1000 Done!